src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +57,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableSring() {
```

Comment:
As discussed in tutorial, you may want to consider the privacy setting when returning a printable string in this system.